storage/mysql/storage_test.go

```diff
@@ -575,6 +575,24 @@ func TestGetLeavesByHashNotPresent(t *testing.T) {
 	}
 }
 
+func TestGetLeavesByLeafValueHashNotPresent(t *testing.T) {
+	logID := createLogID("TestGetLeavesByLeafValueHashNotPresent")
+	s := prepareTestLogStorage(logID, t)
+	tx := beginLogTx(s, t)
+	defer tx.Commit()
+
+	hashes := [][]byte{[]byte("thisdoesn'texist")}
+	leaves, err := tx.GetLeavesByLeafValueHash(hashes, false)
```

Comment:
(style) All of this would normally be folded together:
```go
if leaves, err := tx.GetLeavesByLeafValueHash(hashes, false); err != nil {
    t.Fatalf("Error getting leaves by hash: %v", err)
} else if len(leaves) != 0 {
    t.Fatalf("Expected no leaves returned but got %d", len(leaves))
}
```

Also, Go test error messages usually have a specific format, something like:
```go
if leaves, err := tx.GetLeavesByLeafValueHash(hashes, false); err != nil {
    t.Fatalf("GetLeavesByLeafValueHash(%x)=nil,%v; want [],nil", hashes, err)
} else if len(leaves) != 0 {
    t.Fatalf("GetLeavesByLeafValueHash(%s)=%q,nil; want [],nil", hashes, leaves)
}
```